src/hooks/journals/useJournalData.ts

```diff
@@ -107,28 +129,34 @@ interface UseJournalDataSettings {
     desiredCount?: number;
     maxBytes?: number;
 }
+
 const useJournalData = (
     journalName?: string,
-    collectionName?: string,
-    settings?: UseJournalDataSettings
+    settings?: UseJournalDataSettings,
+    opsJournalTarget?: boolean
 ) => {
     const failures = useRef(0);
     const initialLoadComplete = useRef(false);
 
-    const { data: gatewayConfig } = useGatewayAuthToken(
-        collectionName ? [collectionName] : null
+    const brokerAddress = useJournalStore((state) =>
+        opsJournalTarget
+            ? state.taskBrokerAddress
+            : state.collectionBrokerAddress
+    );
+
+    const brokerToken = useJournalStore((state) =>
+        opsJournalTarget ? state.taskBrokerToken : state.collectionBrokerToken
     );
```

Comment:
Since we are using logic - please combine these two selectors into one. That way it reduces the chance of that logic being wrong if another selector is added.